pkg/webui/components/profile-dropdown/profile-dropdown.styl

```
@@ -39,3 +39,8 @@
 
   &-icon
     margin-right: -.4rem
+
+.profile-picture
+  height: 4rem
+  width: 4rem
```

```suggestion
  height: 3rem
  width: 3rem
```

Let's make it a bit smaller.

---

While we're at it, please also add LL24-31 to:

```stylus
  &:hover
    background-color: $c-backdrop-lighter

  +focus-visible()
    background-color: $c-backdrop-lighter

  &:active, &:focus
    background-color: $c-backdrop
```